tracker-radar-tests/TR-domain-matching/domain_matching_tests.json

```diff
@@ -81,7 +81,10 @@
                 "siteURL": "https://randomsite123.com/",
                 "requestURL": "https://bad.cnames.test/breakage",
                 "requestType": "script",
-                "expectAction": "ignore"
+                "expectAction": "ignore",
+                "exceptPlatforms": [
+                    "ios-browser"
```

Comment:
Oh, why those tests are suddenly failing?

Reply:
I don't think these test cases were tests on iOS when they were added (btw fixed this. There are only 2 new failing tests). I need to sync up with someone on these tests. Idk if there is a regression in iOS or if these are testing something we aren't supporting or a wrong implementation. Regardless I think fixing that is out of scope for this project at the moment but its blocking me with test failures.

Reply:
Yeah agreed, this is an interesting scenario that I'll document. Related fix tasks for those failing tests is https://app.asana.com/0/1200194497630846/1201531896279377/f